applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1006,87 +1052,106 @@ static void stream_configured_cb(struct bt_bap_stream *stream,
 
 	if (dir == BT_AUDIO_DIR_SINK) {
 		/* NOTE: The string below is used by the Nordic CI system */
-		LOG_INF("%s sink stream configured", unicast_servers[device_index].ch_name);
-		le_audio_print_codec(
-			unicast_servers[device_index].cap_sink_stream.bap_stream.codec_cfg, dir);
+		LOG_INF("%s sink stream configured",
```

Comment:
Discuss printing in general. I think we should add CIG %d and CIS %d in several places.

Reply:
As long as we don't support more than one CIG I don't think it makes sense